resources/linux-5.0-config

```diff
@@ -454,6 +454,7 @@ CONFIG_ACPI_HOTPLUG_IOAPIC=y
 # CONFIG_ACPI_HED is not set
 # CONFIG_ACPI_CUSTOM_METHOD is not set
 # CONFIG_ACPI_BGRT is not set
+CONFIG_ACPI_REDUCED_HARDWARE_ONLY=y
```

Comment:
This is indeed needed with the current 5.0 based kernel as the `evged` driver only builds with that option.
The next kernel release (5.5) will have that dependency removed:
https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/drivers/acpi/Makefile#n51

So could we please open an issue tracking the fact that `CONFIG_ACPI_REDUCED_HARDWARE_ONLY=y` should be removed when we switch to 5.5?